### ISSUE TITLE
Importing your own key should be more clear about what formats etc it will accept

### ISSUE THREAD

**Issue by feoh** [keybase/keybase-issues#1571]
I imported a key previously generated with Mailvelope -https://www.mailvelope.com/ and the import process completed successfully.

I have now tracked a bunch of people, but found out that nobody can read messages that I encrypt. They get the error:

Error: The author of this message is unknown to Keybase.

So I'm stuck starting over I guess.

**Comment by andjo** [comment]
I guess I'm having the same issue. Just signed up, and verified a bunch of accounts without problems. Then I tried to send my self a message with the Encrypt + Decrypt and got the message

> Error: the author of this message is unknown to Keybase.

I also happen to use a key generated with Mailvelope, and the email I used there is not the same as I signed up with here. If that is a requirement it might have been helpful to tell med that when I chose to use my own existing key.

**Comment by feoh** [comment]
Yeah I suspect that the key type generated by Mailvelope and what they're
looking for here are incompatible.  Obviously, interoperating with
Mailvelope would be ducky, but at the very least importing the Mailvelope
key should generate an error or warning or something.

On Tue, May 26, 2015 at 4:19 PM, Anders Johansson <notifications@github.com>
wrote:

> I guess I'm having the same issue. Just signed up, and verified a bunch of
> accounts without problems. Then I tried to send my self a message with the
> Encrypt + Decrypt and got the message
>
> Error: the author of this message is unknown to Keybase.
>
> I also happen to use a key generated with Mailvelope, and the email I used
> there is not the same as I signed up with here. If that is a requirement it
> might have been helpful to tell med that when I chose to use my own
> existing key.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/keybase-issues/issues/1571#issuecomment-105653312>
> .
>



-- 
Christopher Patti - Geek At Large | GTalk: cpatti@gmail.com | AIM:
chrisfeohpatti | P: (260) 54PATTI
"Technology challenges art, art inspires technology." - John Lasseter, Pixar
